README.md

```diff
@@ -270,30 +270,65 @@ Need help or want to contribute ideas or code? Head over to our [CONTRIBUTING](C
 
 ## Note for maintainers
 
+* Commits && Releases && Publishing
+
 All the commits in this repo follow the [Conventional Commits spec](https://www.conventionalcommits.org/en/v1.0.0/#summary). When merging a PR, make sure 1/ to
 use squash merge and 2/ that the title of the PR follows the Conventional Commits spec.
 
-The history of commits will be used to generate the `CHANGELOG`. To do so, run `yarn deploy` on the master
-branch. This command will look at all the commits since the latest tag, bump the package version according
-to semver rules, and generate a new `CHANGELOG`.
-
-If you don't want to follow semver or need to do a dry run, consult the [`standard-version` CLI usage](https://github.com/conventional-changelog/standard-version#cli-usag)
-docs. Flags for `standard-version` can be passed to `yarn deploy`.
-
-`yarn deploy`, which only does local operations and doesn't push anything, will output more or
-less the following lines:
-
-``` bash
-$ yarn deploy
-yarn run v1.21.1
-$ yarn build && standard-version -r minor
-$ rimraf lib/ && tsc
-✔ bumping version in package.json from 0.18.1 to 0.18.2
-✔ outputting changes to CHANGELOG.md
-✔ committing package.json and CHANGELOG.md
-✔ tagging release v0.18.2
-ℹ Run `git push --follow-tags origin master && npm publish` to publish
+### Step 1 - Preparation
+
+1. Checkout a branch `name-v5-0-1`. When deciding what version will be released it is important to look over 1) PRs since the last release and 2) release notes for any updated polkadot-js dependencies as they may affect type definitions.
+
+2. Ensure we have the latest polkadot-js dependencies
+
+  * @polkadot/api [release notes](https://github.com/polkadot-js/api/releases)
+  * @polkadot/apps-config [release notes](https://github.com/polkadot-js/apps/releases)
+  * @polkadot/util-crypto [release notes](https://github.com/polkadot-js/common/releases)
+
+  Note: Every monday the polkadot-js ecosystem will usually come out with a new release. It's important that we keep up, and read the release notes for any breaking changes, or high priority updates.
+
+3. After updating the dependencies, the next step is making sure the release will work against all noted runtimes for Polkadot, Kusama, and Westend. This can be handled via the [sidecar-runtime-test](https://github.com/TarikGul/sidecar-runtime-test) helper library. Instructions for how to run it are in the repos README.md. Before moving forward ensure all tests pass, and if it warns of any missing types feel free to make an issue [here](https://github.com/paritytech/substrate-api-sidecar/issues). 
+
+4. Update the version in the package.json (this is very important for releasing on NPM).
+
+5. Update `CHANGELOG.md` by looking at merged PRs since the last release. Follow the format of previous releases. 
+
+  * Make sure to note if it is a high upgrade priority (e.g. it has type definitions for an upcoming runtime upgrade to a Parity maintained network).
```

Comment:
It would be good to keep a checklist of who to notify when we release high priority stuff (likely on to be kept on Notion, but a mention here is good also).